packages/tools/pluggable-widgets-tools/configs/rollup-plugin-collect-dependencies.js

```diff
@@ -16,28 +23,58 @@ export function collectDependencies({ onlyNative, outputDir, widgetName }) {
             managedDependencies.length = 0;
         },
         async resolveId(source, importer) {
-            if (source.startsWith(".") || source.startsWith("/")) {
+            // eslint-disable-next-line no-control-regex
+            const sourceCleanedNullChar = source.replace(/\x00/g, "");
+            if (sourceCleanedNullChar.startsWith(".") || sourceCleanedNullChar.startsWith("/")) {
                 return null;
             }
             const resolvedPackagePath = await resolvePackage(
                 source,
                 dirname(importer ? importer : rollupOptions.input[0])
             );
-            if (resolvedPackagePath && (!onlyNative || (await hasNativeCode(resolvedPackagePath)))) {
-                if (!managedDependencies.includes(resolvedPackagePath)) {
+
+            if (resolvedPackagePath) {
+                const isNotOnlyNativeOrHasNativeCode = !onlyNative || (await hasNativeCode(resolvedPackagePath));
+                if (isNotOnlyNativeOrHasNativeCode && !managedDependencies.includes(resolvedPackagePath)) {
                     managedDependencies.push(resolvedPackagePath);
                 }
-                return { external: true, id: source };
+                if (!dependencies.some(dependency => dependency.packagePath === resolvedPackagePath)) {
+                    dependencies.push({ packagePath: resolvedPackagePath, isTransitive: false });
+                }
+                return isNotOnlyNativeOrHasNativeCode ? { external: true, id: source } : null;
             }
             return null;
         },
+        async generateBundle() {
+            if (!licenseOptions) {
```

Comment:
I dont think `licenseOptions` will ever be falsy. At callsites for `collectDependencies` you are passing an object literal (truthy), and if this isn't the case, you default to an object literal.

Reply:
done

Reply:
This looks the same?

Reply:
Did you change anything?